package.json

```diff
@@ -12,18 +12,26 @@
     "@wordpress/scripts": "^26.19.0",
     "chalk": "^4.1.2",
     "commander": "^9.4.1",
+    "copy-webpack-plugin": "^12.0.2",
     "fast-glob": "^3.3.2",
     "fs-extra": "^11.2.0",
     "husky": "^8.0.2",
     "lint-staged": "^13.1.0",
-    "lodash": "4.17.21"
+    "lodash": "4.17.21",
+    "npm-run-all": "^4.1.5",
+    "webpackbar": "^6.0.1"
   },
   "scripts": {
     "changelog": "./bin/plugin/cli.js changelog",
     "since": "./bin/plugin/cli.js since",
     "readme": "./bin/plugin/cli.js readme",
     "translations": "./bin/plugin/cli.js translations",
-    "build-plugins": "./bin/plugin/cli.js build-plugins",
+    "build-plugins": "npm-run-all 'build:plugin:!(performance-lab)'",
+    "build:plugin:performance-lab": "rm -rf build/performance-lab && mkdir -p build/performance-lab && git archive HEAD | tar -x -C build/performance-lab",
```

Comment:
@felixarntz Added this command, in case we ever need to confirm what's being added in the build by deploy action.

Reply:
@felixarntz I believe that we should use the above command to verify distribution files(maybe just before a release?). I can see that the `build-cs` directory is being committed in the svn at https://plugins.trac.wordpress.org/browser/performance-lab/trunk for the most current release 😞. 

Also, it would be good if we log an issue to remove such redundant files/dirs from all our plugins in the next release.